pages/migration/index.rst

```diff
@@ -12,6 +12,14 @@ Migration
 This chapter describes which migrations are performed when upgrading your Nuts node. Most of them will be automatic, but sometimes
 manual action is required.
 
+0.14
+^^^^
+Starting version 0.14 vendors and organizations will have an X.509 certificate (encoded in the JWK) associated with their key pairs.
+These certificates are used to identify the holder of the key pair used to sign events, which is also introduced in this version.
+
+This migration can be performed by the Nuts node automatically; use the newly introduced `registry verify` command with the `--fix` flag
```

Comment:
No clear indication that this 'fix' is mandarory and if stuff will break > 0.14

Reply:
fixed